File: Python/003/003.py
import math

"""
*	Problem #: 003
*	Date Attempted: 05/28/20
*	Date Solved: 05/28/20
*
*	What is the largest prime factor of the number 600851475143 ?
*
*	Answer: 6857
"""


def findLargestPrimeNumber(long):
    again = True
    while again:
        again = False
        for i in range(2, int(math.sqrt(long))):
            if long % i == 0:
                print(str(long) + " / " + str(i))
                long = long / i
                again = True
                break
    return long


print(findLargestPrimeNumber(600851475143))

File: Java/003/_003.java

/*
*	Problem #: 003
*	Date Attempted: 05/28/20
*	Date Solved: 05/29/20
*
*	What is the largest prime factor of the number 600851475143 ?
*
*	Answer: 6857
*/

public class _003 {
    public static void main(String[] args) {
        boolean again = true;
        long input = 600851475143L;
        while (again) {
            again = false;
            for (int i = 2; i < Math.sqrt(input); i++) {
                if (input % i == 0) {
                    again = true;
                    input /= i;
                }
            }
        }

        System.out.println(input);
    }
}